clang/include/clang/AST/DeclBase.h

```diff
@@ -2711,6 +2715,12 @@ class DeclContext {
                    bool Deserialize = false) const;
 
 private:
+  /// Lookup all external visible declarations and the external declarations
+  /// within the same module specified by \param NamedModule. We can't
```

Comment:
Should we replace `\param` here (and other places around the patch) with `\c` (or `\ref`)? IIUC `\param` is just used to start a parameter description.

Reply:
Done